microservices-connector/cmd/router/main.go

```diff
@@ -741,185 +1159,45 @@ func handleMultipartError(writer *multipart.Writer, err error) {
 	log.Error(err, "Error during multipart creation")
 }
 
-// create a handler to handle traffic to /ui
-// if the payload is empty, redirect to ui service
-// if there's payload, format the payload and redirect to backend service
-func mcUiHandler(w http.ResponseWriter, req *http.Request) {
```

Comment:
> **Router logic changes**
>
> Hanlding guard 466 specific http code to stop processing pipeline and still result body

Um, what about the removal of this large UI handling function, and the next asset handling one?